--- /storage/emulated/0/build.gradle.kts.bak	2021-08-15 09:42:08.661007864 +0800
+++ /storage/emulated/0/build.gradle.kts	2021-08-15 09:42:47.901007849 +0800
@@ -15,7 +15,7 @@
         consumerProguardFiles("proguard-rules.pro")
 
         externalNativeBuild.ndkBuild {
-            abiFilters("armeabi-v7a", "arm64-v8a", "x86", "x86_64")
+            abiFilters("armeabi-v7a", "arm64-v8a")
             arguments("-j${Runtime.getRuntime().availableProcessors()}")
         }
 
@@ -35,7 +35,7 @@
 cargo {
     module = "src/main/rust/shadowsocks-rust"
     libname = "sslocal"
-    targets = listOf("arm", "arm64", "x86", "x86_64")
+    targets = listOf("arm", "arm64")
     profile = findProperty("CARGO_PROFILE")?.toString() ?: currentFlavor
     extraCargoBuildArguments = listOf("--bin", libname!!)
     featureSpec.noDefaultBut(arrayOf(
