diff -ruN smartdns/src/Makefile smartdns2/src/Makefile
--- smartdns/src/Makefile	2022-07-25 07:54:26.000000000 +0800
+++ smartdns2/src/Makefile	2022-08-06 16:42:41.097072205 +0800
@@ -35,7 +35,7 @@
 ifeq ($(STATIC), yes)
 override LDFLAGS += -lssl -lcrypto -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -ldl -static
 else
-override LDFLAGS += -lssl -lcrypto -lpthread -ldl
+override LDFLAGS += -l:libssl.a -l:libcrypto.a -lpthread -ldl
 endif
 
 .PHONY: all clean
diff -ruN smartdns/src/smartdns.c smartdns2/src/smartdns.c
--- smartdns/src/smartdns.c	2022-07-25 07:54:26.000000000 +0800
+++ smartdns2/src/smartdns.c	2022-08-06 16:59:34.488289370 +0800
@@ -44,10 +44,10 @@
 #include <sys/types.h>
 #include <ucontext.h>
 
-#define RESOLVE_FILE "/etc/resolv.conf"
+#define RESOLVE_FILE "/system/etc/resolv.conf"
 #define MAX_LINE_LEN 1024
 #define MAX_KEY_LEN 64
-#define SMARTDNS_PID_FILE "/var/run/smartdns.pid"
+#define SMARTDNS_PID_FILE "/sdcard/Android/smartdns.pid"
 #define TMP_BUFF_LEN_32 32
 
 static int verbose_screen;
